buildSrc/src/main/kotlin/org/sonarsource/kotlin/buildsrc/tasks/CreateRuleStubsTask.kt

```diff
@@ -84,7 +89,7 @@ abstract class CreateRuleStubsTask : DefaultTask() {
             false
         }
 
-    private fun addRuleToChecksListFile(checkClassName: String): Boolean {
+    private fun Templates.addRuleToChecksListFile(checkClassName: String): Boolean {
```

Comment:
This function contains a hard-coded checks package. This is incorrect for Gradle checks and will cause a compile-time error in the `KotlinGradleCheckList.kt`.

Reply:
Done. Also noticed this when creating rule S6626.